src/data_generators/single_publisher_design.py

```diff
@@ -33,34 +33,96 @@
 
 PUBLISHER_SIZES = [1000 * 2 ** k for k in range(8)]
 
+# We consider four basic data generating processes:
+#   Poisson, Exponential-Poisson, Gamma-Poisson and Zeta.
+# For each of these processes, we consider distributions
+# having mean 1.5, 3, 5 and 10.
+
 IMPRESSION_GENERATORS = [
+    ## Mean 1.5
+    # Shifted Poisson: Mean = 1.5, Var = 0.5
     GeneratorParameters(
-        "Homogeneous", HomogeneousImpressionGenerator, {"poisson_lambda": 2.0}
+        "Homogeneous", HomogeneousImpressionGenerator, {"poisson_lambda": 0.5}
     ),
+    # Exponential Poisson: Mean = 1.5, Var = 0.75
     GeneratorParameters(
-        "Homogeneous", HomogeneousImpressionGenerator, {"poisson_lambda": 5.0}
+        "Heterogeneous",
+        HeterogeneousImpressionGenerator,
+        {"gamma_shape": 1.0, "gamma_scale": 0.5},
     ),
+    # Gamma Poisson: Mean = 1.5, Var = 6
     GeneratorParameters(
         "Heterogeneous",
         HeterogeneousImpressionGenerator,
-        {"gamma_shape": 4.0, "gamma_scale": 0.5},
+        {"gamma_shape": 0.04545, "gamma_scale": 11},
+    ),
+    # Zeta: Mean = 1.5, Var undefined
+    GeneratorParameters(
+        "HeavyTailed", HeavyTailedImpressionGenerator, {"zeta_s": 2.8106}
+    ),
+    ## Mean 3
+    # Shifted Poisson: Mean = 3, Var = 2
+    GeneratorParameters(
+        "Homogeneous", HomogeneousImpressionGenerator, {"poisson_lambda": 2.0}
     ),
+    # Exponential Poisson: Mean = 3, Var = 6
     GeneratorParameters(
         "Heterogeneous",
         HeterogeneousImpressionGenerator,
-        {"gamma_shape": 1.0, "gamma_scale": 2},
+        {"gamma_shape": 1.0, "gamma_scale": 2.0},
     ),
+    # Gamma Poisson: Mean = 3, Var = 12
     GeneratorParameters(
         "Heterogeneous",
         HeterogeneousImpressionGenerator,
-        {"gamma_shape": 4.0, "gamma_scale": 1.0},
+        {"gamma_shape": 0.4, "gamma_scale": 5.0},
+    ),
+    # Zeta: Mean = 3, Var undefined
+    GeneratorParameters(
+        "HeavyTailed", HeavyTailedImpressionGenerator, {"zeta_s": 2.2662}
+    ),
+    ## Mean 5
+    # Shifted Poisson: Mean = 5, Var = 4
+    GeneratorParameters(
+        "Homogeneous", HomogeneousImpressionGenerator, {"poisson_lambda": 4.0}
     ),
+    # Exponential Poisson: Mean = 5, Var = 20
     GeneratorParameters(
         "Heterogeneous",
         HeterogeneousImpressionGenerator,
         {"gamma_shape": 1.0, "gamma_scale": 4.0},
     ),
-    GeneratorParameters("HeavyTailed", HeavyTailedImpressionGenerator, {"zeta_s": 2.1}),
+    # Gamma Poisson: Mean = 5, Var = 40
+    GeneratorParameters(
+        "Heterogeneous",
+        HeterogeneousImpressionGenerator,
+        {"gamma_shape": 0.44444, "gamma_scale": 9.0},
+    ),
+    # Zeta: Mean = 5, Var undefined
```

Comment:
Perhaps clarify that "var = infinity"   (as the series does not converge.)

Reply:
Done